addons/hr_holidays/models/resource.py

```diff
@@ -64,6 +63,7 @@ def _reevaluate_leaves(self, time_domain_dict):
             'state': 'draft',
         })
         self.env.add_to_compute(self.env['hr.leave']._fields['number_of_days'], leaves)
```

Comment:
I'm wondering if this line is needed then, as the `duration_display` field depends on it :thinking: Did you try the same test as the one added below but with hours, to see if `number_of_hours` is also correctly computed?

Reply:
Tried removing the line, and doesn't seem like the ORM doesn't recompute `number_of_days` automatically when computing `duration_display`.

Expanded the tests to also check `duration_display` for `hr.leave.type`s using `hour` as `request_unit`, seems to work as expected.

Reply:
Thanks for checking!